{
  "metadata": {
    "@context": "https://openvex.dev/ns/v0.2.0",
    "@id": "https://github.com/canonical/ubuntu-security-notices/blob/main/vex/cve/2024/CVE-2024-10096",
    "author": "Canonical Ltd.",
    "timestamp": "2025-07-14T04:55:54.391099",
    "version": 1
  },
  "statements": [
    {
      "vulnerability": {
        "@id": "https://nvd.nist.gov/vuln/detail/CVE-2024-10096",
        "name": "CVE-2024-10096",
        "description": "Rejected reason: This CVE ID has been rejected or withdrawn by its CVE Numbering Authority.",
        "aliases": [
          "https://www.cve.org/CVERecord?id=CVE-2024-10096",
          "https://huntr.com/bounties/a4be847b-a52d-42cc-9e78-3299e2d30ab2",
          "https://ubuntu.com/security/CVE-2024-10096"
        ]
      },
      "timestamp": "2025-03-20 10:15:00 UTC",
      "products": [
        {
          "@id": "pkg:deb/ubuntu/dask.distributed@1.25.0+ds.1-1?arch=source&distro=focal"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@1.25.0+ds.1-1?arch=amd64"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@1.25.0+ds.1-1?arch=arm64"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@1.25.0+ds.1-1?arch=armhf"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@1.25.0+ds.1-1?arch=i386"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@1.25.0+ds.1-1?arch=ppc64el"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@1.25.0+ds.1-1?arch=riscv64"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@1.25.0+ds.1-1?arch=s390x"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@1.25.0+ds.1-1?arch=amd64"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@1.25.0+ds.1-1?arch=arm64"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@1.25.0+ds.1-1?arch=armhf"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@1.25.0+ds.1-1?arch=i386"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@1.25.0+ds.1-1?arch=ppc64el"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@1.25.0+ds.1-1?arch=riscv64"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@1.25.0+ds.1-1?arch=s390x"
        }
      ],
      "status": "affected",
      "action_statement": "This package (for the given release) is no longer supported. Please upgrade your system."
    },
    {
      "vulnerability": {
        "@id": "https://nvd.nist.gov/vuln/detail/CVE-2024-10096",
        "name": "CVE-2024-10096",
        "description": "Rejected reason: This CVE ID has been rejected or withdrawn by its CVE Numbering Authority.",
        "aliases": [
          "https://www.cve.org/CVERecord?id=CVE-2024-10096",
          "https://huntr.com/bounties/a4be847b-a52d-42cc-9e78-3299e2d30ab2",
          "https://ubuntu.com/security/CVE-2024-10096"
        ]
      },
      "timestamp": "2025-03-20 10:15:00 UTC",
      "products": [
        {
          "@id": "pkg:deb/ubuntu/dask.distributed@2022.01.0+ds.1-1?arch=source&distro=jammy"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2022.01.0+ds.1-1?arch=amd64"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2022.01.0+ds.1-1?arch=arm64"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2022.01.0+ds.1-1?arch=armhf"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2022.01.0+ds.1-1?arch=i386"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2022.01.0+ds.1-1?arch=ppc64el"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2022.01.0+ds.1-1?arch=riscv64"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2022.01.0+ds.1-1?arch=s390x"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2022.01.0+ds.1-1?arch=amd64"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2022.01.0+ds.1-1?arch=arm64"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2022.01.0+ds.1-1?arch=armhf"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2022.01.0+ds.1-1?arch=i386"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2022.01.0+ds.1-1?arch=ppc64el"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2022.01.0+ds.1-1?arch=riscv64"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2022.01.0+ds.1-1?arch=s390x"
        },
        {
          "@id": "pkg:deb/ubuntu/dask.distributed@2023.12.1+ds-4?arch=source&distro=noble"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2023.12.1+ds-4?arch=amd64"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2023.12.1+ds-4?arch=arm64"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2023.12.1+ds-4?arch=armhf"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2023.12.1+ds-4?arch=i386"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2023.12.1+ds-4?arch=ppc64el"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2023.12.1+ds-4?arch=riscv64"
        },
        {
          "@id": "pkg:deb/ubuntu/python-distributed-doc@2023.12.1+ds-4?arch=s390x"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2023.12.1+ds-4?arch=amd64"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2023.12.1+ds-4?arch=arm64"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2023.12.1+ds-4?arch=armhf"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2023.12.1+ds-4?arch=i386"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2023.12.1+ds-4?arch=ppc64el"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2023.12.1+ds-4?arch=riscv64"
        },
        {
          "@id": "pkg:deb/ubuntu/python3-distributed@2023.12.1+ds-4?arch=s390x"
        }
      ],
      "status": "under_investigation",
      "status_notes": "The vulnerability of this package (for the given release) is not known and needs to be evaluated."
    }
  ]
}